selfcal_helpers.py

```diff
@@ -1619,14 +1648,19 @@ def get_max_uvdist(vislist,bands,band_properties):
          baselines=get_baseline_dist(vis)
          all_baselines=np.append(all_baselines,baselines)
       max_baseline=np.max(all_baselines)
+      min_baseline=np.min(all_baselines)
+      baseline_5=numpy.percentile(all_baselines,5.0)
       baseline_75=numpy.percentile(all_baselines,75.0)
       baseline_median=numpy.percentile(all_baselines,50.0)
       for vis in vislist:
          meanlam=3.0e8/band_properties[vis][band]['meanfreq']
          max_uv_dist=max_baseline # leave maxuv in meters like the other uv entries /meanlam/1000.0
+         min_uv_dist=min_baseline
          band_properties[vis][band]['maxuv']=max_uv_dist
+         band_properties[vis][band]['minuv']=max_uv_dist
```

Comment:
should this be min_uv_dist and not max_uv_dist?

Reply:
Indeed it should, though I don't think it is actually used anywhere. Still, I'll fix that up

Reply:
Should be fixed now